app/javascript/react/components/App.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch } from "react-router-dom"
+
 import BrandsIndexContainer from "./BrandsIndexContainer"
 import BrandShowContainer from "./BrandShowContainer"
-import { BrowserRouter, Route, Switch } from "react-router-dom"
+import PizzaShowContainer from "./PizzaShowContainer"
+
 
 const App = (props) => {
   return (
     <BrowserRouter>
       <Switch>
+        <Route exact path="/" component={BrandsIndexContainer}/>
         <Route exact path="/pizzas" component={BrandsIndexContainer}/>
         <Route exact path="/pizzas/:id" component={BrandShowContainer}/>
+        <Route exact path="/pizza/:id" component={PizzaShowContainer}/>
```

Comment:
unfortunately, this is poor routing that breaks with established convention

I would
- update line 15 to be `/brands/:id` 
- line 16 to be `/pizzas/:id`